doc/Reference/Markup/StaticAndThemeResources.md

```diff
@@ -0,0 +1,158 @@
+
+### Creating a new Resource
+
+Many times we may want to create resources, but we also want to be able to easily reference them later. Let's start by looking at a case where we need to define a path string that we will use later for an icon.
+
+```cs
+public static class MyResources
+{
+    public static readonly Resource<Geometry> MyIcon =
+        StaticResource.Create<Geometry>(nameof(MyIcon), "{Path String}");
+}
+```
+
+Looking at this sample we actually are achieving multiple things. One is that we are able to more easily make use of our path string as we can provide it the generic argument for a `Geometry` object which is what will be expected when we use it. We also are able to avoid trying to figure out how to convert it. 
+
+Lastly, in the case of WinUI we get the advantage that this will be created as a `Geometry` object each time we need it, this will keep it from being parented and prevent it from being reused later. 
+
+Now to use our `Resource<Geometry>`, we can simply reference it similar to our constant string, except that we need to also add it to a `ResourceDictionary` somewhere in the Visual Tree.
+
+```csharp
+public partial class MyPage : Page
+{
+    public MyPage()
+    {
+        this.Resources(r => r.Add(MyResources.MyIcon))
+            .Content(
+                new PathIcon()
+                    .Data(MyResources.MyIcon)
+            );
+    }
+}
+```
+
+Now let's look at the equivalent code of what this is in effect doing:
+
+```csharp
+public partial class MyPage : Page
+{
+    public MyPage()
+    {
+        this.Resources(r => r.Add<Geometry>("MyIcon", "{path string}"))
+            .Content(
+                new PathIcon()
+                    .Data(x => x.StaticResource("MyIcon"))
+            );
+    }
+}
+```
+
+### Getting an Existing Resource
+
+Similarly, we may not want to use the lambda directly. In this case we can again use the `StaticResource` type to get our resource. This can also be helpful for resources that you may not get as often and which may be added for you already to the Resources of a 3rd Party `ResourceDictionary` you have brought in.
+
+```csharp
+new PathIcon()
+    .Data(StaticResource.Get<Geometry>("MyIcon"))
+```
+
+### Strongly Typed Keys
+
+Along this same path we may want to use a resource frequently, but we don't need to handle creating the resource or adding it to the Resource Dictionary. In this case we might want to instead use the `StaticResourceKey<T>` similar to our `Resource<T>`. In this case we would simply update our `MyResources` class like follows:
+
+```csharp
+public static class MyResources
+{
+    public static readonly StaticResourceKey<Geometry> MyIcon = new StaticResourceKey<Geometry>(nameof(MyIcon));
+}
+```
+
+Since we do not need to add it to the Resource Dictionary as it may have come from a Merged Dictionary higher up in the Visual Tree we now might have code that simply looks like:
```

Comment:
```suggestion
### Getting an Existing Resource

Similarly, we may not want to use the lambda directly. In this case, we can again use the `StaticResource` type to get our resource. This can also be helpful for resources that you may not get as often and which may be added for you already to the Resources of a 3rd Party `ResourceDictionary` you have brought in.

```csharp
new PathIcon()
    .Data(StaticResource.Get<Geometry>("MyIcon"))
```

### Strongly Typed Keys

Along this same path, we may want to use a resource frequently, but we don't need to handle creating the resource or adding it to the Resource Dictionary. In this case, we might want to instead use the `StaticResourceKey<T>` similar to our `Resource<T>`. In this case, we would simply update our `MyResources` class like follows:

```csharp
public static class MyResources
{
    public static readonly StaticResourceKey<Geometry> MyIcon = new StaticResourceKey<Geometry>(nameof(MyIcon));
}
```

Since we do not need to add it to the Resource Dictionary as it may have come from a Merged Dictionary higher up in the Visual Tree we now might have code that simply looks like this:
```